ChangeLog

```diff
@@ -1,6 +1,18 @@
 ChangeLog for S3FS
 ------------------
 
+Version 1.92 -- 21 May, 2023 (major changes only)
```

Comment:
The stream upload feature will not make it in time for 1.91 and will be included in 1.92, and about credlib, so please add:
```
#1802 - #2104 - New option: streamupload
#1927 - #2101 - New option: credlib and credlib_ops
```

Reply:
Done, sorry I missed the first one and the second one was unclear before.

Reply:
Confirmed the fix.